docs/diagrams/RemoveActivitDiagram.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
```

Comment:
filename wrongly spelled, missing a "y"

Reply:
it's intended, i can't add y there otherwise it's not puml file, to see whether you can add y on your end

Reply:
done